nyc-turnstile-counts/lookup/stations.csv

```diff
@@ -510,3 +510,5 @@
 509,NULL,"R550","","PTH","","Hoboken","","","","40.7349102","-74.0332199","",""
 510,NULL,"R551","","PTH","","Grove St","","","","40.7236214","-74.0448325","",""
 511,NULL,"R552","","PTH","","Journal Sq","","","","40.7303043","-74.0686325","",""
+512,NULL,"R535","","JFK","","Howard Beach","","","","40.661051","-73.829638","",""
+513,NULL,"R552","","JFK","","Jamaica","","","","40.698942","-73.808416","",""

```

Comment:
Oops, just realized that the `complex_id` for this should be `R536`. Forgot to replace it in the copy-and-paste.